test/karma.conf.js

```diff
@@ -113,7 +119,7 @@ module.exports = function (config) {
 
     // Continuous Integration mode
     // if true, Karma captures browsers, runs the tests and exits
-    singleRun: false,
+    singleRun: true,
```

Comment:
I'd like to keep `singleRun` as `false` since we change this option via `npm scripts` when running CI.

Reply:
Yes, a test forgotten. It's now fixed.